hack/otel/otelcol.yaml

```
@@ -0,0 +1,21 @@
+# Receive signals over gRPC and HTTP
+# moby currently uses http
+receivers:
+  otlp:
+    protocols:
+      grpc:
+      http:
```

Just pulled `otel/opentelemetry-collector:latest` and it now defaults to `localhost`, so we need this instead:

```suggestion
      grpc:
        endpoint: 0.0.0.0:4317
      http:
        endpoint: 0.0.0.0:4318
```